mne/source_estimate.py

```diff
@@ -1894,18 +1894,26 @@ def save(self, fname, ftype="stc", *, overwrite=False, verbose=None):
             spaces are obtained by adding ``"-lh.stc"`` and ``"-rh.stc"`` (or
             ``"-lh.w"`` and ``"-rh.w"``) to the stem provided, for the left and
             the right hemisphere, respectively.
-        ftype : str
-            File format to use. Allowed values are ``"stc"`` (default),
-            ``"w"``, and ``"h5"``. The ``"w"`` format only supports a single
-            time point.
+        ftype : str | None
```

Comment:
the suggestion in the linked issue was to use `"auto"` (not `None`) for the "infer-from-fname" behavior. I don't have very strong feelings one way or another, but @larsoner might?

Reply:
you changed the logic below, but didn't change this line.
```suggestion
        ftype : "auto" | "stc" | "w" | "h5"
```